recipes-ni/ni-systemimage/files/nisystemimage

```diff
@@ -5,8 +5,6 @@
 set -eEu -o pipefail
 shopt -s dotglob nullglob
 
-. /etc/natinst/networking/functions.common
```

Comment:
Why is this unnecessary? Is it unused?

Reply:
Yes, I didn't see any of the symbols from `/etc/natinst/networking/functions.common` being used here.